core/controllers/access_validators.py

```diff
@@ -39,20 +37,25 @@ class ClassroomAccessValidationHandler(base.BaseHandler):
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
-    URL_PATH_ARGS_SCHEMAS = {} # type: Dict[Text, Any]
-    HANDLER_ARGS_SCHEMAS = {
+    # Type[str, Any] is used to match the type defined for this attribute in
+    # its parent class `base.BaseHandler`.
+    URL_PATH_ARGS_SCHEMAS: Dict[str, Any] = {}
+    # Type[str, Any] is used to match the type defined for this attribute in
+    # its parent class `base.BaseHandler`.
+    HANDLER_ARGS_SCHEMAS: Dict[str, Any] = {
         'GET': {
             'classroom_url_fragment': {
                 'schema': {
                     'type': 'basestring'
                 }
             }
         }
-    } # type: Dict[Text, Any]
+    }
 
+    # Using type ignore[misc] here because untyped decorator makes function
+    # "get" also untyped.
     @acl_decorators.open_access # type: ignore[misc]
-    def get(self):
-        # type: () -> None
+    def get(self) -> None:
         classroom_url_fragment = self.normalized_request.get( # type: ignore[union-attr]
```

Comment:
I think this ignore needs a written justification (basically, in general, anything that's not `no-untyped-call`).

Reply:
Actually this is because the file `core/controllers/base.py` isn't fully typed yet. I've added a short comment here indicating the action to be performed. I had already I left a comment where we define this attribute in BaseHandlers (the parent of this class) - [here](https://github.com/oppia/oppia/blob/79a3d387b7ac931629360496fd714cb7a2d297fc/core/controllers/base.py#L184-L189) and I am keeping it too - it will help others understand the situation if similar cases are encountered at other places until base.py is typed.

Once type annotations are added and type casting is used, mypy will itself throw error for this ignore saying it's unwanted. I hope that would help us get rid of this ignore at the right time.